src/openforms/tests/e2e/test_input_validation.py

```diff
@@ -670,3 +678,60 @@ def test_regex_pattern(self):
             ui_input="<h2>test</h2>",
             expected_ui_error="Ongeldig Nederlands kenteken",
         )
+
+
+class SingleFileTests(ValidationsTestCase):
```

Comment:
also needs tests for:

* trying to upload a `.pdf` file that isn't a PDF
* trying to upload a file with an extension that isn't in the allowed file types